express.js

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * ./express.js
+ *
+ * Set up the express server.
+ */
+
+/* eslint-disable global-require */
+/* eslint-disable no-process-env */
+const conf = require('./config');
+const signal = require('./signal/signal');
+const ONE = 1;
+const featureToggles = require('feature-toggles');
+const helmet = require('helmet');
+const swaggerTools = require('swagger-tools');
+const errorHandler = require('./api/v1/errorHandler');
+const path = require('path');
+const fs = require('fs');
+const yaml = require('js-yaml');
+const ipfilter = require('express-ipfilter');
+const rejectMultipleXForwardedFor =
+  require('./config/rejectMultipleXForwardedFor');
+const bodyParser = require('body-parser');
+const env = conf.environment[conf.nodeEnv];
+const ENCODING = 'utf8';
+const compress = require('compression');
+const cors = require('cors');
+const etag = require('etag');
+
+// set up server side socket.io and redis publisher
```

Comment:
`redis publisher`? not being set up here anymore, right?

Reply:
Also maybe just move this comment up to line 12

Reply:
The sample store initialization was moved, the pub/sub is still here because it's dependent on the express app.

Reply:
Though maybe it would make sense to extract that as well, and maybe also socket.io.
My initial approach was to keep everything that had a dependency on the express app in this file, which is why I kept these here. But then I had to break that rule and use the exported app object for the processId because it was only available in that context. So I could probably do the same thing for redis and socket.io, as long as the tests don't rely on them.